bcp/client/src/webpack/helper/toGeoJSON.js

```diff
@@ -0,0 +1,154 @@
+import currentLocationJSON from '../../../../public/currentLocations.json';
```

Comment:
I'm a bit confused why these .json files are being imported, surely there should be a store where the json files are pulled down from the server.